docs/docs/core/useAnimatedStyle.mdx

```diff
@@ -58,6 +60,12 @@ Only relevant when using Reanimated [without the Babel plugin on the Web](https:
 
 `useAnimatedStyle` returns an animated style object which has to be passed to the `style` property of an [Animated component](/docs/fundamentals/glossary#animated-component) that you want to animate.
 
+Types of `useAnimatedStyle` are made so they mimic behavior of `StyleSheet` styles as much as possible - `updater` callback return value should just look like a regular style object - with the exception of using shared values.
+
+:::caution
+For better compliance with custom components, `useAnimatedStyle` return value type might seem like it can be appended to a non-animated component - but this will result in an error. Always use `useAnimatedStyle` return value with animated components only.
```

Comment:
Please rewrite it in active voice and simplify it. It's definitely written too in a too complicated fashion than it needs to be